onnxruntime/core/providers/nnapi/nnapi_builtin/builders/op_support_checker.h

```diff
@@ -4,18 +4,21 @@
 #pragma once
 
 #include "helper.h"
+#include "qdq_support_helper.h"
 
 namespace onnxruntime {
 namespace nnapi {
 
 struct OpSupportCheckParams {
-  OpSupportCheckParams(int32_t android_feature_level, bool use_nchw)
+  OpSupportCheckParams(int32_t android_feature_level, bool use_nchw, std::unique_ptr<QDQSupportHelper> qdq_support_helper)
       : android_feature_level(android_feature_level),
```

Comment:
if we don't do any initialization of qdq_support checker before we pass it in it could just be a simple member (no unique_ptr required)